container.go

```diff
@@ -96,7 +96,7 @@ type Container interface {
 	//
 	// errors:
 	// Systemerror - System error.
-	Set() error
+	Set(config configs.Config) error
```

Comment:
Yes, I was intend to make Set more functionality, not just change resource configs. We can reusing Config in libcontainer, and leave it to Docker to decide which Config can be changed.